config/local.yaml

```diff
@@ -2,3 +2,5 @@ application:
   host: 0.0.0.0
   subscribed:
     redirect: https://example.com
+  mailer: !SendGrid
+    token: "none"
```

Comment:
How would we set this via env? I view these config files as more of a default that can be overridden by env.

Reply:
Hmmm... Good question. That might be hard to do via ENV. Especially if we have pretty complicated settings for anything other than token. Let me go back to the drawing board a bit and push up another commit

Reply:
We might be able to set this while still setting `!SendGrid` in the yml... But I don't think there'd be a way to override the type here in the ENV